src/Hadolint/Formatter/Format.hs

```diff
@@ -46,13 +44,14 @@ toResult res =
     Left err -> Result (Seq.singleton err) mempty
     Right c -> Result mempty (Seq.fromList (sort c))
 
-severityText :: Severity -> String
+severityText :: Maybe Severity -> String
 severityText s =
   case s of
-    ErrorC -> "error"
-    WarningC -> "warning"
-    InfoC -> "info"
-    StyleC -> "style"
+    Just ErrorC -> "error"
+    Just WarningC -> "warning"
+    Just InfoC -> "info"
+    Just StyleC -> "style"
+    Nothing -> "ignore"
```

Comment:
Will this be printed somewhere? It would be weird to see `ignore`d stuff

Reply:
No. Rules with a severity of `Nothing` will not produce any output. But it would be weird to have a non-exhaustive match here and `ignore` is the most sane return value I could come up with.